SergiiDotsenko/src/library/Reader.java

```diff
@@ -3,7 +3,7 @@
 public class Reader {
     private int readerID;
     private static int totalReaders = 0;
-    private final static int maxBooksOnHands = 3;
+    private final static int maxBooksOnHands = 3;//Подумать какие константы можно добавить в вашу программу "Библиотека", при наличии таких - реализуйте вашу задумку.
```

Comment:
final static it's constant, so instead of maxBooksOnHands make variable MAX_BOOKS_ON_HANDS